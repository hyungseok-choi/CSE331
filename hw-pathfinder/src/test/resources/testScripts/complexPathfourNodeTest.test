CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D

AddEdge simpleGraph A B 1.0
AddEdge simpleGraph A B 2.0
AddEdge simpleGraph A C 7.0
AddEdge simpleGraph A C 9.0
AddEdge simpleGraph A D 3.0
AddEdge simpleGraph B D 3.0
AddEdge simpleGraph B D 5.0
AddEdge simpleGraph D C 3.0
AddEdge simpleGraph D C 4.0

FindPath simpleGraph A C
