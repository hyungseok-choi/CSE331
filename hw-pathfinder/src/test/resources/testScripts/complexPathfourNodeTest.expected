created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph

added edge 1.000 from A to B in simpleGraph
added edge 2.000 from A to B in simpleGraph
added edge 7.000 from A to C in simpleGraph
added edge 9.000 from A to C in simpleGraph
added edge 3.000 from A to D in simpleGraph
added edge 3.000 from B to D in simpleGraph
added edge 5.000 from B to D in simpleGraph
added edge 3.000 from D to C in simpleGraph
added edge 4.000 from D to C in simpleGraph

path from A to C:
A to D with weight 3.000
D to C with weight 3.000
total cost: 6.000
